modules/meteroid/src/config.rs

```diff
@@ -80,6 +81,9 @@ pub struct Config {
 
     #[envconfig(nested)]
     pub mailer: MailerConfig,
+
+    #[envconfig(nested)]
+    pub oauth: OauthConfig,
```

Comment:
could this be optional ? (to allow local dev without requiring external api keys)

Reply:
i will need to make the internal fields optional. You can just copy the fake clientid/clientsecret values from the env.example so you don't need to specify the real values

Reply:
alright :+1: